### DISCUSSION TITLE
Writing direction (`rtl`) missing in PHP for languages

### DISCUSSION THREAD

**Discussion by JonathanWebNL** [annexare/Countries#145] [Q&A] [answered]
I'm trying to import the data from this package using composer. In the SQL files I see a rtl column for the languages, but this field seems to be missing in the PHP json import. Am I missing something or should I make a feature request out of this?

**Comment by dmythro** [comment, marked as answer]
Hi @JonathanWebNL, `rtl` is optional, only a few languages have it, when it is truthy. Otherwise it doesn't exist for the most of languages. Maybe it is the case?

**Reply by JonathanWebNL** [reply]
Ah thanks, I expected the RTL to be always filled with 0 or 1. But this will do fine:)

**Reply by dmythro** [reply]
It simply parses a JSON under the hood. Making it optional saves some space, reduces JSON size :)
